src/pages/Home/utils/getUserById.ts

```diff
@@ -0,0 +1,31 @@
+import { User } from 'src/utils';
+import { useRoute, useRouter } from 'vue-router';
+import { getSpecificUser } from 'src/actions';
+import { ref } from 'vue';
+import { Notify } from 'quasar';
+
+export const getUserbyId = async () => {
+  const user = ref<User>();
+  const route = useRoute();
+  const id = 'b5ca33ca-fd87-4858-a285-e15429c6dd9b';
```

Comment:
Why is this hardcoded?

@e-azocar please check

Reply:
I was testing an implementation of a function to bring the data of a user. in this task it is not related but the function can be used later if you want I can remove it or I can take it to userActions.

Reply:
@Eghinner @e-azocar please check

Reply:
@AdrianFagilde please check this comment https://github.com/Azordev/did-app/pull/71#discussion_r962342631